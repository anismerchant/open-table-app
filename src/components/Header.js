import React from 'react'
import { PageHeader } from 'antd'

const styles = {
  pageHeaderContainer: {
    border: '1px solid #D9D9D9',
    marginBottom: 10,
  }
}

export default () => (
  <PageHeader 
    style={styles.pageHeaderContainer} 
    title="OpenTable" 
    subTitle="Find restaurants near you" 
  />
)
