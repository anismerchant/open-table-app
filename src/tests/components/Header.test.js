import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

test('It should correctly render Header.', () => {
  const wrapper = shallow(<Header  />);
  expect(wrapper).toMatchSnapshot();
});
