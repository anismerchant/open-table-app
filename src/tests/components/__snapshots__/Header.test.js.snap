// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should correctly render Header. 1`] = `
<PageHeader
  backIcon={
    <Icon
      type="arrow-left"
    />
  }
  style={
    Object {
      "border": "1px solid #D9D9D9",
      "marginBottom": 10,
    }
  }
  subTitle="Find restaurants near you"
  title="OpenTable"
/>
`;
